### DISCUSSION TITLE
When is it planned to support table driven method per scenario?

### DISCUSSION THREAD

**Discussion by brennankoopman** [getgauge/gauge-vscode#833] [Ideas]
As from the gauge website, it states the extension still does not support this feature. It there a planned release for this?

![image](https://user-images.githubusercontent.com/29217829/148806056-ae943628-82d8-4016-b704-7cca41e6567c.png)
https://docs.gauge.org/writing-specifications.html?os=windows&language=javascript&ide=vscode

**Comment by sriv** [comment]
It is marked experimental because there isn't support for authoring intellisense (you may even see this being reported as a parse error in some editors). 

The other major work is on the reports end, where currently the reporting plugins are not capable of accounting for scenario tables.

There is no timeline for this as of now, because there is no ongoing work on this unfortunately. Happy to hear thoughts/feedback and if there's any interest in contributing towards this do let me know!

**Comment by jensakejohansson** [comment]
This feature (table driven scenarios) has been worked on in Gauge and is not longer in an "experimental state". It took a while, but better late then never...
